ImportExport/Export2Slicer.FCMacro

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Opens current visible objects in Slic3r. One can use different slicing software
+
+__Title__ = "Export to Slicer"
+__Author__ = "Damian Łoziński"
+__Version__ = "0.1"
+__Date__ = "2020-04-26"
+__Comment__ = "Export selected objects to amf/stl files and open it in Slicer"
+__Web__ = "https://forum.freecadweb.org/viewtopic.php?f=10&t=4686&sid=e69e70bc34c1987e215b5fc6fcb6b367&start=20"
+__Wiki__ = ""
+__Icon__ = "Export2Slicer.png"
+__Help__ = "Run this macro to selected objects in slicer software. You may need to change slicer path"
+__Status__ = ""
+__Requires__ = "freecad 0.17+"
+__Communication__ = "lozinski dot d at gmail dot com"
+__Files__ = "Export2Slicer.png"
+
+import os
+import subprocess
+import Mesh
```

Comment:
Add
```
import FreeCAD as app
import FreeCADGui as gui
```
And use `app` and `gui` in place of `App` and `Gui`.

Reply:
ok